src/test/rgw/test_multi.py

```diff
@@ -159,7 +159,7 @@ def init(parse_args):
     try:
         path = os.environ['RGW_MULTI_TEST_CONF']
     except KeyError:
-        path = tpath('test_multi.conf')
+        path = os.path.join('test_multi.conf')
```

Comment:
oops, i had removed `tpath()` in 55f276ba535980be3f93c0d5d1303ad1ae17a84e:
```py
def tpath(bin, *params):
    s = test_path + bin
    for p in params:
        s += ' ' + str(p)

    return s
```
so i think this should be `path = test_path + 'test_multi.conf'`